Kitodo-XML-SchemaConverter/pom.xml

```diff
@@ -43,6 +43,7 @@
         <dependency>
             <groupId>net.sf.saxon</groupId>
             <artifactId>Saxon-HE</artifactId>
+            <version>${saxon.version}</version>
```

Comment:
This should not necessary as the used version of a package is controlled in the global pom file.